### PR TITLE
fix published npm package + version bump + opt goerli + gitignore generated files

### PR DESCRIPTION
**Pull request by oveddan**
* Fix how package is published to npm - by linking to proper main file
* Slight version bump to allow a new package to be published
* Update chain configs for new chains
* Add opt goerli addresses to wagmi generated config
* gitignore generated files

### CODE REVIEW

File: package/chainConfigs.ts [hunk 1]
```diff
@@ -9,6 +9,11 @@ export const chainConfigs = {
     MINT_FEE_AMOUNT: 777000000000000,
     MINT_FEE_RECIPIENT: "0x7A810DCd0f8d83B20212326813Db6EF7E9FD030c",
   },
+  [11155111]: {
```
Comment (oveddan) [inline package/chainConfigs.ts:12]: maybe we should git ignore these files, and they are just built and bundled at publish time? same for the wagmi generated file

File: src/version/ContractVersionBase.sol [hunk 2]
```diff
@@ -8,6 +8,6 @@ import {IVersionedContract} from "../interfaces/IVersionedContract.sol";
 contract ContractVersionBase is IVersionedContract {
     /// @notice The version of the contract
     function contractVersion() external pure override returns (string memory) {
-        return "1.3.0";
+        return "1.3.1";
```
Comment (oveddan) [inline src/version/ContractVersionBase.sol:11]: i wonder if this should always have to match the package version - what if we are just publishing js changes to npm?

File: package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "@zoralabs/zora-1155-contracts",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "repository": "git@github.com:ourzora/creator-contracts.git",
   "author": "Iain <iain@zora.co>",
   "license": "MIT",
-  "main": "./dist/index.js",
+  "main": "./dist/index.cjs",
```
Comment (oveddan) [inline package.json:7]: this will fix the package import from npm

File: package/chainConfigs.ts (hunk 1 above)
Comment (iainnash) [reply package/chainConfigs.ts:12]: yep good idea

Comment (oveddan) [reply package/chainConfigs.ts]: done!

**Review by iainnash — APPROVED** [review]
lgtm, thanks for cleaning this up!

**Pull request merged**